cha/implementaion/googlesample.tex

```diff
@@ -0,0 +1,36 @@
+\section{Foundation of Our Android App}
+This section describes our implementation and adaptation of a sample app by Google called \textit{UniversalMusicPlayer}\footnote{https://github.com/googlesamples/android-UniversalMusicPlayer} as a foundation for our app.
+We use this sample such that we can focus our project on the device communication and psychoacoustic effects from the beginning, rather than building a generic Android music player with suitable \ac{UI}.
+The sample is a large codebase, as such we do not cover everything, but rather we present the elements and general implementation choices.
+The code provided by Google follows the android development guidelines, as such this section will introduce the basic android building blocks that we use in further development of the application.
+
+These basic building blocks of an Android application are: Activities; services; and intents.
```

Comment:
Activities and services are both described below, this listing gives the reader an idea of relation, as such one would expect a description of how/why we use intents since activities and services are described in a way as to explain why we use services ---> Intents should be mentioned/described in some fashion if it is to be part of this list.